android-client-sdk/build.gradle

```diff
@@ -30,6 +32,12 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
+
+        // Debug build type to be used for local development & unit tests
+        debug {
+            minifyEnabled true
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
```

Comment:
I dont think we want to enable minify at all except for test. This would make it so our public listeners etc are automatically minified if testing on debug. 

if a client is trying to make a modification and builds a debug AAR, things such as "onUpdate" would be obfuscated.

Reply:
cool will do... :) and yeah, there are not other dependencies that we need